acceptance/tests/reports/corrective_change_new_resource.rb

```diff
@@ -0,0 +1,83 @@
+require 'yaml'
+
+test_name "C98092: corrective change new resource" do
+require 'puppet/acceptance/environment_utils'
+extend Puppet::Acceptance::EnvironmentUtils
+
+  app_type = File.basename(__FILE__, '.*')
```

Comment:
The name "app_type" isn't self-descriptive. I think it was used in the existing test code by mistake. It should be changed to something more self-descriptive. I think this variable stores the name of this test file... so possibly rename this to "test_file_name"?